docs/sql_statements/data_definition_statements.md

```diff
@@ -457,7 +457,14 @@ Query OK, 0 rows affected (0.01 sec)
 
 `Syntax`
 ```
-  DROP INDEX index_name ON table_name
+DROP INDEX index_name ON tbl_name
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/drop-index.html

Reply:
> Be consistent with the documentation. https://dev.mysql.com/doc/refman/8.0/en/drop-index.html

The same question as above.